src/pages/CertificatesViewAll.js

```diff
@@ -1,69 +1,93 @@
+}
 
 export default function CertificatesViewAll() {
   const { current } = useContext(Context)
+  const { data, loading, error, callApiFn: fetch } = useAxios()
+  const navigate = useNavigate()
+
   const persona = personas.find(({ id }) => id === current)
+  const url = `${persona.origin}/v1/certificate`
+
+  React.useEffect(() => {
+    if (!data) fetch({ url })
+  }, [data, fetch, url])
+
   return (
     <>
       <Nav />
-      <Header userFullName={persona.name} companyName={persona.company} />
-      <LeftWrapper area="timeline"></LeftWrapper>
-      <MainWrapper>
-        <Grid.Panel area="main">
-          <Container>
-            <Text>
-              Certificates View All: <br />
-              Certificate One <br />
-              Certificate Two <br />
-              ...
-            </Text>
-          </Container>
-        </Grid.Panel>
-        <Sidebar area="sidebar"></Sidebar>
-      </MainWrapper>
+      <Header
+        userFullName={persona.name}
+        companyName={error || persona.company}
+      />
+      <Timeline area="timeline"></Timeline>
+      <Main area="main">
+        {(loading || error) && (
+          <Spinner
+            text={error || 'loading...'}
+            size={'large'}
+            color={persona.background}
+          />
+        )}
+        {data && (
+          <Table
+            action={(item) =>
+              navigate(`/certificate/${item.original_token_id}`)
+            }
+            headers={headersMap[current]}
+            rows={data.map((cert) => [
+              /* TODO with other cert viewing stories
+                  - create rows/cels per persona using "headersMap as an example"*/
+              <NameCell
+                key={cert.id}
+                date={new Date(cert.created_at).toISOString()}
+                name={`SQNC-HYPROOF-${cert.original_token_id.toString().padStart(4, '0')}`}
+              />,
+              `${cert.hydrogen_quantity_wh}`,
+              `${cert.energy_consumed_wh}`,
+              `${cert.embodied_co2}`,
+              stateToStatus[(cert.embodied_co2 && 'co2') || cert.state],
+            ])}
+            variant="hyproof"
+          />
+        )}
+        {data?.length === 0 && 'nothing to render'}
+      </Main>
     </>
   )
 }
 
-const LeftWrapper = styled(Grid.Panel)`
-  max-width: 400px;
-  max-height: 100%;
-  padding: 20px 0px;
-  overflow: hidden;
-  background: #0c3b38;
+const Timeline = styled(Grid.Panel)`
+  background: rgb(39, 132, 122);
 `
 
-const MainWrapper = styled.div`
-  display: grid;
-  grid: subgrid / subgrid;
-  grid-area: 2 / 1 / -1 / -1;
-  overflow: hidden;
-  text-align: center;
-`
-
-const Sidebar = styled(Grid.Panel)`
-  align-items: center;
-  justify-items: center;
-  min-width: 340px;
-  color: white;
-  background: #0c3b38;
-`
-
-const Container = styled.div`
-  display: grid;
+const Main = styled(Grid.Panel)`
+  background: rgb(39, 132, 122);
+  color: #fff;
+  width: 100%;
+  align-self: center;
```

Comment:
I don't know where to add this .. hopefully you don't mind if I add it here .. 

the top right triangle doesn't seem to want to render properly .. ( maybe it doesn't want to render on tables or tds )

<img width="1079" alt="image" src="https://github.com/digicatapult/sqnc-hyproof-client/assets/58742260/d96b0ea8-b0a2-422c-b733-6014aacff994">

Here , this is the original tutorial i've used , if that helps : https://nicolasgallagher.com/pure-css-folded-corner-effect/

EDIT: I'm using my screen from my laptop .. no external monitor or anything like that ..

Reply:
ugh this is annoying. @n3op2 can you fix this in your next PR please